### DISCUSSION TITLE
How about multiple domains?

### DISCUSSION THREAD

**Discussion by billsey** [fire1ce/cloudflareDDNS-PowerShell#13] [Q&A]
I have several different domains pointing at my home web server. I'd like to update all of them with one run of the script, since they will all end up with the same DNS record. Would it work to use something like a comma delimited list for $dns_record to provide the list of A records to update?

**Comment by cyb233** [comment]
My fork already supports it, you can add any A/AAAA record as needed ( But I haven't tested "@" yet, maybe you can help me test that? )

the conf will like:
```powershell
$records = @(
    @{
        type       = "A"
        what_ip    = "internal"
        dns_record = "ddns.example.com"
        ttl        = 60
        comment    = ""
        proxied    = $false
    },
    @{
        type       = "AAAA"
        what_ip    = "internal"
        dns_record = "ddns.example.com"
        ttl        = 60
        comment    = ""
        proxied    = $false
    }
)
```
